test/Common/DurableTaskEndToEndTests.cs

```diff
@@ -2146,6 +2146,44 @@ public async Task Purge_Partially_History_By_TimePeriod(bool extendedSessions)
             }
         }
 
+        /// <summary>
+        /// End-to-end test which validates that incorrect task hub name throws instance of <see cref="ArgumentException"/>.
+        /// </summary>
+        [Theory]
+        [Trait("Category", PlatformSpecificHelpers.TestCategory)]
+        [Trait("Category", PlatformSpecificHelpers.TestCategory + "_BVT")]
+        [InlineData("Task-Hub-Name-Test")]
+        [InlineData("1TaskHubNameTest")]
+        [InlineData("/TaskHubNameTest")]
+        [InlineData("-taskhubnametest")]
+        [InlineData("taskhubnametesttaskhubnametesttaskhubnametesttaskhubnametesttaskhubnametesttaskhubnametest")]
+        public async Task TaskHubName_Thows_ArgumentException(string taskHubName)
+        {
+            await this.VerifyExceptionOnBadInputForTaskHubName(taskHubName);
+        }
+
+        private async Task VerifyExceptionOnBadInputForTaskHubName(string taskHubName)

```

Comment:
Much better! Can you remove this private method and put all the logic directly inside `TaskHubName_Thows_ArgumentException`?

Reply:
Hello @cgillum,

Thank you for this optimization!

I will fix it.

Thank you!